src/config/db.ts

```diff
@@ -0,0 +1,12 @@
+import mongoose from 'mongoose';
+
+export async function connectToDatabase() {
+  await mongoose.connect(
+    'mongodb+srv://oleg:32156@clusternodejs.cpbapnx.mongodb.net/moviesdb?retryWrites=true&w=majority',
```

Comment:
Better to move sensitive data to .env file and do not commit it to git.

Reply:
Thank you very much for pointing out how important it is not to commit such data into Git.
I was going to use dotenv, but I didn’t know how and to whom to pass environment variables so that when someone checks my homework, they can connect to the database.
I would be very grateful if you write how this is usually done in practice when checking homework or in other cases